llvm/utils/TableGen/VTEmitter.cpp

```diff
@@ -29,6 +32,54 @@ class VTEmitter {
 
 } // End anonymous namespace.
 
+std::string VTtoGetLLVMTyString(const Record *VT) {
```

Comment:
static. Could also avoid the temp string if you just directly write to a passed in ostream

Reply:
Updated, Twine and std::string both are drop now

Reply:
Moved into anon namespace

Reply:
I though LLVM coding standards say that only classes should be in anonymous namespace? https://llvm.org/docs/CodingStandards.html#anonymous-namespaces

Reply:
@topperc Oh yeah, you're right.

Reply:
Reverted that change :P